backend/api/v1/audit.go

```diff
@@ -37,7 +39,7 @@ func NewAuditInterceptor(store *store.Store) *AuditInterceptor {
 	}
 }
 
-func createAuditLog(ctx context.Context, request, response any, method string, storage *store.Store, rerr error) error {
+func createAuditLog(ctx context.Context, request, response any, method string, storage *store.Store, deltas any, rerr error) error {
```

Comment:
deltas -> we shouldn't pass in deltas. We should pass in serviceData instead. Making it generic instead of specific.